src/main/route/UserRouter.ts

```diff
@@ -0,0 +1,21 @@
+import express, { Router } from 'express';
+import { UserController } from '../controller/UserController';
+
+export class UserRouter {
+  private userController: UserController;
+  private userRouter = express.Router();
+
+  constructor(userController: UserController) {
+    this.userController = userController;
+  }
+
+  getRoutes(): Router {
+    this.userRouter.get('/:userId', this.userController.getUser);
+    this.userRouter.get(
+      '/:userId/savedExercises',
+      this.userController.getSavedExercises
```

Comment:
Since `userId` is always an integer, maybe we can also include a condition that ensures the given userId is an integer using regex. I believe adding `(\\d+)` after the `:userid` will route to the function only if the given parameter is an integer. 

More info: https://expressjs.com/en/guide/routing.html